packages/app/src/components/search/SearchPage.tsx

```diff
@@ -84,13 +92,36 @@ const SearchPage = () => {
                 ]}
               />
               <Paper className={classes.filters}>
+                {types.includes('techdocs') && (
+                  <SearchFilter.Autocomplete
+                    className={classes.filter}
+                    label="Entity"
+                    name="name"
+                    asyncValues={async partial => {
+                      // Return a list of entitis which are documented.
```

Comment:
```suggestion
                      // Return a list of entities which are documented.
```

Reply:
Haha and again ... re-fetching every single time? If so, probably wrap in a component that loads them all once on mount or so?